notebooks/enterprise-search/app-search-engine-exporter.ipynb

```diff
@@ -0,0 +1,894 @@
+        "          \"en-stop-words-filter\",\n",
+        "          \"en-stem-filter\"\n",
+        "        ],\n",
+        "        \"tokenizer\": \"whitespace\"\n",
+        "      },\n",
+        "      \"q_prefix\": {\n",
+        "        \"filter\": [\n",
+        "          \"cjk_width\",\n",
+        "          \"lowercase\",\n",
+        "          \"asciifolding\"\n",
+        "        ],\n",
+        "        \"tokenizer\": \"standard\"\n",
+        "      },\n",
+        "      \"i_text_base\": {\n",
+        "        \"filter\": [\n",
+        "          \"cjk_width\",\n",
+        "          \"lowercase\",\n",
+        "          \"asciifolding\",\n",
+        "          \"en-stop-words-filter\"\n",
+        "        ],\n",
+        "        \"tokenizer\": \"standard\"\n",
+        "      },\n",
+        "      \"q_text_base\": {\n",
+        "        \"filter\": [\n",
+        "          \"cjk_width\",\n",
+        "          \"lowercase\",\n",
+        "          \"asciifolding\",\n",
+        "          \"en-stop-words-filter\",\n",
+        "          \"synonyms-filter\"\n",
+        "        ],\n",
+        "        \"tokenizer\": \"standard\"\n",
+        "      },\n",
+        "      \"iq_text_stem\": {\n",
+        "        \"filter\": [\n",
+        "          \"cjk_width\",\n",
+        "          \"lowercase\",\n",
+        "          \"asciifolding\",\n",
+        "          \"en-stop-words-filter\",\n",
+        "          \"en-stem-filter\",\n",
+        "        ],\n",
+        "        \"tokenizer\": \"standard\"\n",
+        "      },\n",
+        "      \"i_text_bigram\": {\n",
+        "        \"filter\": [\n",
+        "          \"cjk_width\",\n",
+        "          \"lowercase\",\n",
+        "          \"asciifolding\",\n",
+        "          \"en-stem-filter\",\n",
+        "          \"bigram_joiner\",\n",
+        "          \"bigram_max_size\"\n",
+        "        ],\n",
+        "        \"tokenizer\": \"standard\"\n",
+        "      },\n",
+        "      \"q_text_bigram\": {\n",
+        "        \"filter\": [\n",
+        "          \"cjk_width\",\n",
+        "          \"lowercase\",\n",
+        "          \"asciifolding\",\n",
+        "          \"synonyms-filter\",\n",
+        "          \"en-stem-filter\",\n",
+        "          \"bigram_joiner_unigrams\",\n",
+        "          \"bigram_max_size\"\n",
+        "        ],\n",
+        "        \"tokenizer\": \"standard\"\n",
+        "      }\n",
+        "    }\n",
+        "  }\n",
+        "}\n",
+        "\n",
+        "# and actually create our index\n",
+        "elasticsearch.indices.create(\n",
+        "  index= DEST_INDEX,\n",
+        "  mappings={\n",
+        "    \"properties\": mapping\n",
+        "  },\n",
+        "  settings=settings\n",
+        ")"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "# Add `sparse_vector` fields for semantic search\n",
```

Comment:
We should probably call this out as an optional in the section header, additive step, and link to some documentation on semantic search? Could also be a good call out to link to search labs here. 

It also requires the prereq of an ML node